xml/deployment_intro.xml

```diff
@@ -69,8 +69,9 @@
    </listitem>
   </varlistentry>
  </variablelist>
- <xi:include href="common_intro_target_audience_i.xml"/>
  <xi:include href="common_intro_available_doc_i.xml"/>
  <xi:include href="common_intro_feedback_i.xml"/>
  <xi:include href="common_intro_typografie_i.xml"/>
+ <xi:include href="common_intro_making_i.xml"/>
+ <xi:include href="common_intro_lifecycle_support_i.xml"/>
```

Comment:
Nope, the openSUSE books have separate intros where the new snippet about support and life cycle is not included. 

I also don't see any issues with having the 'making' snippet in the SLE book intros (the snippet was included in one them anyway, but not in the others, so I tried to make it consistent by adding it to all book intros). @fsundermeyer : What do you think?

Reply:
I think I railed pretty hard against the making intro once which is probably the reason it is excluded now.

This intro tends to be incorrect (past iterations were mentioning XEP when we used both FOP and XEP, etc.; even the current version only mentions PDF builds but not HTML or EPUB).
It is also not particularly useful for the average customer. It would be much better to imo to expand a bit on the "you can contribute" section in "Feedback" and have some sort of contributor's guide (or just the repo README) linked from there

Reply:
I agree with your POV, fixed with c31343a20 and 5bfafeb012a16.